CHANGELOG.md

```diff
@@ -27,12 +28,60 @@ fn update(&mut self, _ctx: &mut Context) -> GameResult {
 ```
 This will print `Coroutine says: "I came from a coroutine!"` after 100 frames.
 
+### 3d
+
+You can now do 3d rendering with bevy. This entails a lot of new api ground to cover. Here are the main points.
```

Comment:
Assuming `bevy` is a typo here, I'd rewrite this to be a little more concise and avoid "now", which doesn't have a reference.

> ggez is capable of basic 3d rendering. The APIs and types differ from 2d in the following ways.
> `Canvas3d` and `Drawable3d` are the 3d equivalents of `Canvas` and `Drawable`. A simple example of drawing might look like this: